hangman/test/hangman_test.exs

```diff
@@ -0,0 +1,147 @@
+defmodule HangmanTest do
+  use ExUnit.Case
+  doctest Hangman
+
+  test "create new game" do
+    assert Hangman.new_game().letters != []
+  end
+
```

Comment:
A good set of tests.

However, you might have made them a little simpler, and removed some duplication, by changing the definition of
`new_game` to

~~~ elixir
 def new_game(word \\ random_word()) do
    %Hangman.Game.State{letters: word}
  end
~~~

That way you could pass in the word, and then use the regular API to get to the state you wanted. This would end up testing more of your code.